sphinx/source/docs/user_guide/webgl.rst

```diff
@@ -27,10 +27,38 @@ Support
 -------
 
 Only a subset of Bokeh's objects are capable of rendering in WebGL. Currently
-supported are the circle and line glyphs, and many markers: asterisk, circle,
-square, diamond, triangle, inverted_triangle, cross, circle_cross, square_cross,
-diamond_cross, x, square_x, and circle_x. You can safely combine multiple glyphs
-in a plot, even if some are rendered in WebGL, and some are not.
+supported are the circle and :class:`~bokeh.models.glyphs.Line` glyphs, and many markers:
```

Comment:
How about simplifying both sentences with:
> Bokeh's WebGL support covers a subset of glyphs. This includes the :class:`~bokeh.models.glyphs.Line` glyph, and most markers:

Reply:
Sounds good! Just pushed up latest updates. Let me know what you think!